src/ArborX_DistributedTree.hpp

```diff
@@ -153,14 +153,25 @@ DistributedTree<MemorySpace, Enable>::DistributedTree(
                          "ArborX::DistributedTree::DistributedTree::"
                          "rank_bounding_boxes"),
       comm_size);
-  // FIXME when we move to MPI with CUDA-aware support, we will not need to
-  // copy from the device to the host
+#ifdef ARBORX_USE_CUDA_AWARE_MPI
+  Box bottom_tree_bounds;
+  if (!_bottom_tree.empty())
+    Details::expand(bottom_tree_bounds, _bottom_tree.bounds());
```

Comment:
Why is it not just unconditional assignment?

Reply:
Daniel was trying to avoid data transfer in his original PR.
Did you try wrapping the root bounding volume in an unmanaged view like he was doing?
Here is how you'd get the pointer to it on the device
https://github.com/arborx/ArborX/blob/5b28f1089bd3ba38f5dea5796daa0dfc9221d45c/src/ArborX_LinearBVH.hpp#L121-L130

Reply:
(Note that I am not sure which is faster)

Reply:
> Why is it not just unconditional assignment?

Could do. But I thought it would be clearer this way, because otherwise we have to guarantee that the bounding box of any search index is invalid. I don't know if we make this promise.

> Daniel was trying to avoid data transfer in his original PR.

Yes, you can do that. How would you use it with other search indexes, though? For example, lets say we want to use BruteForce as the bottom tree. It does not have a concept of root.

Reply:
> > Why is it not just unconditional assignment?
> 
> Could do. But I thought it would be clearer this way, because otherwise we have to guarantee that the bounding box of any search index is invalid. I don't know if we make this promise.

It is not clearer and yes assignment is guaranteed to work.


> > Daniel was trying to avoid data transfer in his original PR.
> 
> Yes, you can do that. How would you use it with other search indexes, though? For example, lets say we want to use BruteForce as the bottom tree. It does not have a concept of root.

Good point about other acceleration data structure.  Also I realize now on HIP or Cuda, both versions imply a kernel launch in local memory and I expect it will more or less take the same time since all we will measure is launch latency.

Reply:
> It is not clearer and yes assignment is guaranteed to work.

I meant, do we promise that if a hierarchy is empty, it's bounding box is invalid. It is in implementation, but I don't think semantically we do.

Reply:
I don't understand the "kernel launch in local memory" part.